libavformat/dshow.h

```diff
@@ -0,0 +1,168 @@
+/*
+* DShow Structure Definition Copies
+* Copyright (c) 2017 Dsessions
```

Comment:
@derek-se I think I should change this to a Panopto copyright.  What do you think?